web/src/lib/components/photos-page/asset-grid.svelte

```diff
@@ -202,12 +205,17 @@
 
   let shiftKeyIsDown = false;
 
+  const deselectAllAssets = () => {
+    $isSelectingAllAssets = false;
+    assetInteractionStore.clearMultiselect();
+  };
+
   const onKeyDown = (event: KeyboardEvent) => {
     if ($isSearchEnabled) {
       return;
     }
 
-    if (matchesShortcut(event, { key: 'Shift', shift: true })) {
+    if (event.key === 'Shift') {
```

Comment:
Why do we reverting back to the previous implementation here?

Reply:
I asked that already and it handles edge cases better: https://github.com/immich-app/immich/pull/8105#discussion_r1532625879